JS/Lesson31/onder/my-app/src/components/createTask/İndex.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import './style.css';
+
+export const CreateTask = ({ onCreateProps }) => {
+
+  const [task, setTask] = useState("");
+
+  const handleInputChange = (event) => {
+    setTask(event.target.value);
+  }
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (task.length > 0) {
+      onCreateProps(task);
+      setTask("");
+    }
+  };
+
+  return (
+    <>
+      <h2>Create task </h2>
+     
+      <form className="form" onSubmit={handleSubmit}>
+      <button type="submit" className="create-btn">Save</button>
+        <button onClick={Submit} className="create-btn">Save</button>
```

Comment:
This button does not seem to be correct, the one above should work just fine. 
The button should have type "submit" but we don't want `onClick` event here. Instead, we add the `onSubmit` event to the `form` itself so that both press on `Enter` and on this button (as it has "submit" type) would trigger the function that is passed to the `onSubmit` event listener (`handleSubmit`).